elyra/pipeline/kfp/component_parser_kfp.py

```diff
@@ -28,9 +28,10 @@
 
 
 class KfpComponentParser(ComponentParser):
```

Comment:
Because parsers are _contributed_ and not _byo_'d, we should move the KFP and AA parsers in the parent directory.

Reply:
After discussion with @kevin-bates, we're going to leave these files where they are for now